tensorflow/lite/kernels/concatenation_test.cc

```diff
@@ -26,9 +26,24 @@ using ::testing::ElementsAreArray;
 
 class BaseConcatenationOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, axis, input
-  // dimensions.
   BaseConcatenationOpModel() {}
+  BaseConcatenationOpModel(
```

Comment:
Rather than duplicating all the code under BaseConcatenationOpModel, you can reuse the one below and provide a default with fused_activation_function = ActivationFunctionType_NONE.

Reply:
@alanchiao , thanks for the review i have updated the code as per your suggestion, kindly check.

Regards
Amit